sympy/polys/euclidtools.py

```diff
@@ -1248,6 +1248,579 @@ def dup_zz_heu_gcd(f, g, K):
     raise HeuristicGCDFailed('no luck')
 
 
+def sparse_coeffs(p1, syms):
+    """
+    It converts a polynomial p1 into a sparse representation where the
+    monomials are split into parts with symbols and parts without symbols,
+    and the coefficients are stored accordingly.
+
+    Parameters
+    ==========
+        p1 : sympy.Poly
+            The polynomial to compute the sparse coefficients for.
+        syms : set
+            The set of symbols to keep in the sparse coefficients.
+
+    Returns
+    ==========
+        dict
+            A dictionary of sparse coefficients. The keys are tuples of the symbols
```

Comment:
The indentation here is off and the number of `=` signs should match the number of characters in Returns.

Reply:
See the style guide:
https://docs.sympy.org/latest/contributing/docstring.html